security/advancedtls/examples/go.mod

```diff
@@ -9,7 +9,6 @@ require (
 )
 
 require (
-	github.com/golang/protobuf v1.5.3 // indirect
```

Comment:
How are these deps changing without any changes under this sub module?

Reply:
This is an indirect dep, so it changes because the grpc deps change.